pkg/skaffold/deploy/kustomize.go

```diff
@@ -100,9 +101,13 @@ type KustomizeDeployer struct {
 	BuildArgs          []string
 	globalConfig       string
 	addSkaffoldLabels  bool
+	useKubectl         bool
 }
 
 func NewKustomizeDeployer(runCtx *runcontext.RunContext) *KustomizeDeployer {
+	// if user has kustomize binary, prioritize that over kubectl kustomize
+	useKubectl := !kustomizeBinaryCheck()
```

Comment:
Also need to check that [kubectl version ≥ 1.14](https://github.com/kubernetes-sigs/kustomize#kubectl-integration) too

Reply:
good catch